libs/community/langchain_community/document_loaders/directory.py

```diff
@@ -51,6 +52,7 @@ def __init__(
             path: Path to directory.
             glob: Glob pattern to use to find files. Defaults to "**/[!.]*"
                (all files except hidden).
+            exclude: patterns to exclude from results, use glob syntax
```

Comment:
Could you provide an example of how to exclude something?

Reply:
@eyurtsev this is a direct copy from the docstring in the file system loader: https://github.com/langchain-ai/langchain/blob/e10030e2417bfdf540bb2e528a8b301cf6377b95/libs/community/langchain_community/document_loaders/blob_loaders/file_system.py#L68

Should I update all occurrences to keep them consistent?

Reply:
I added some docstrings examples based on other loaders.